fw/http_stream.h

```diff
@@ -264,4 +319,29 @@ tfw_h2_stream_fsm_ignore_err(TfwH2Ctx *ctx, TfwStream *stream,
 	return tfw_h2_stream_fsm(ctx, stream, type, flags, true, &err);
 }
 
+static inline u64
+tfw_h2_stream_default_deficit(TfwStream *stream)
+{
+	return 65536 / stream->weight;
```

Comment:
Today we discussed that it could be hard to observe any performance improvements of our website loading time (e.g. LCP) with better streams prioritization. But I made more research on this https://github.com/tempesta-tech/tempesta/issues/1411#issuecomment-2146187050 - see that the 2 crucial for LCP `EXCLUSIVE` streams share bandwidth - probably this is the problem of our current (in `master`) logic of round-robin scheduling and this PR shows different behavior.

Please test this and do more research in the area.